src/borg/fuse.py

```diff
@@ -223,6 +223,7 @@ def __init__(self, key, manifest, repository, args, decrypted_repository):
         # made up and are not contained in the archives. For example archive directories or intermediate directories
         # not contained in archives.
         self._items = {}
+        self._read_cache = LRUCache(capacity=1, dispose=lambda _: None)
```

Comment:
guess we'll need capacity > 1 if parallel reading of more than a file should perform better?

Reply:
Yes